CONTRIBUTING.MD

```diff
@@ -141,3 +141,10 @@ or
 ```
 npm run styleguide:build
 ```
+
+## Releasing a new version of Snacks
+1. `npm run build_release`
```

Comment:
why not `release:build` and `release:publish` to be consistent w/ `test:lint`?

Reply:
Good point 👍